### PR TITLE
kde/gear: 22.04.0 -> 22.04.1

### LINKED ISSUE #172139: calligra fails to build
### Describe the bug

```
error: builder for '/nix/store/6zz5p3x7gpc3gd689nfh0q8ggg25g5qm-calligra-3.2.1.drv' failed with exit code 2;
       last 10 log lines:
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/SheetPrint_p.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/odf/SheetsOdfValidity.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/odf/SheetsOdfRegion.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/odf/SheetsOdfCell.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/Value.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/ValidityStorage.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/Validity.cpp.o
       > [ 67%] Linking CXX shared library libcalligrasheetsodf.so
       > [ 67%] Built target calligrasheetsodf
       > make: *** [Makefile:136: all] Error 2
       For full logs, run 'nix log /nix/store/6zz5p3x7gpc3gd689nfh0q8ggg25g5qm-calligra-3.2.1.drv'.
```

https://hydra.nixos.org/jobset/nixpkgs/trunk#tabs-jobs

Screenshot from 2022-05-09 09-09-34

### Steps To Reproduce
1. ...
2. ...
3. ...

### Expected behavior
no fail

### Notify maintainers
@ebzzry @zraexy

### Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.34, NixOS, 22.05 (Quokka), 22.05pre375061.c777cdf5c56`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.0`
 - channels(root): `"home-manager-21.11.tar.gz, nixos, nixos-hardware, nixos-unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

### PR DESCRIPTION
**Pull request by andrevmatos**
###### Description of changes
Bump KDE Gear / Applications to [22.04.1, released in 11 May 2022](https://kde.org/announcements/gear/22.04.1/)
Also, includes calligra fix for poopler 22.04 : fixes #172139

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by andrevmatos** [comment]
Result of `nixpkgs-review pr 172733` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>337 packages marked as broken and skipped:</summary>
  <ul>
    <li>kexi</li>
    <li>libsForQt512.akonadi</li>
    <li>libsForQt512.akonadi-calendar</li>
    <li>libsForQt512.akonadi-calendar-tools</li>
    <li>libsForQt512.akonadi-contacts</li>
    <li>libsForQt512.akonadi-import-wizard</li>
    <li>libsForQt512.akonadi-mime</li>
    <li>libsForQt512.akonadi-notes</li>
    <li>libsForQt512.akonadi-search</li>
    <li>libsForQt512.akonadiconsole</li>
    <li>libsForQt512.akregator</li>
    <li>libsForQt512.angelfish</li>
    <li>libsForQt512.ark</li>
    <li>libsForQt512.baloo-widgets</li>
    <li>libsForQt512.bomber</li>
    <li>libsForQt512.bovo</li>
    <li>libsForQt512.calendarsupport</li>
    <li>libsForQt512.dolphin</li>
    <li>libsForQt512.dolphin-plugins</li>
    <li>libsForQt512.dragon</li>
    <li>libsForQt512.elisa</li>
    <li>libsForQt512.eventviews</li>
    <li>libsForQt512.falkon</li>
    <li>libsForQt512.ffmpegthumbs</li>
    <li>libsForQt512.filelight</li>
    <li>libsForQt512.granatier</li>
    <li>libsForQt512.grantleetheme</li>
    <li>libsForQt512.gwenview</li>
    <li>libsForQt512.incidenceeditor</li>
    <li>libsForQt512.k3b</li>
    <li>libsForQt512.kaccounts-integration</li>
    <li>libsForQt512.kaccounts-providers</li>
    <li>libsForQt512.kaddressbook</li>
    <li>libsForQt512.kalarm</li>
    <li>libsForQt512.kalendar</li>
    <li>libsForQt512.kalzium</li>
    <li>libsForQt512.kamoso</li>
    <li>libsForQt512.kapman</li>
    <li>libsForQt512.kapptemplate</li>
    <li>libsForQt512.kate</li>
    <li>libsForQt512.katomic</li>
    <li>libsForQt512.kblackbox</li>
    <li>libsForQt512.kblocks</li>
    <li>libsForQt512.kbounce</li>
    <li>libsForQt512.kbreakout</li>
    <li>libsForQt512.kcachegrind</li>
    <li>libsForQt512.kcalc</li>
    <li>libsForQt512.kcalutils</li>
    <li>libsForQt512.kcharselect</li>
    <li>libsForQt512.kcolorchooser</li>
    <li>libsForQt512.kdebugsettings</li>
    <li>libsForQt512.kdeconnect-kde</li>
    <li>libsForQt512.kdegraphics-mobipocket</li>
    <li>libsForQt512.kdegraphics-thumbnailers</li>
    <li>libsForQt512.kdenetwork-filesharing</li>
    <li>libsForQt512.kdenlive</li>
    <li>libsForQt512.kdepim-addons</li>
    <li>libsForQt512.kdepim-runtime</li>
    <li>libsForQt512.kdeplasma-addons</li>
    <li>libsForQt512.kdf</li>
    <li>libsForQt512.kdialog</li>
    <li>libsForQt512.kdiamond</li>
    <li>libsForQt512.keditbookmarks</li>
    <li>libsForQt512.kfind</li>
    <li>libsForQt512.kfloppy</li>
    <li>libsForQt512.kgeography</li>
    <li>libsForQt512.kget</li>
    <li>libsForQt512.kgpg</li>
    <li>libsForQt512.khelpcenter</li>
    <li>libsForQt512.kidentitymanagement</li>
    <li>libsForQt512.kig</li>
    <li>libsForQt512.kigo</li>
    <li>libsForQt512.killbots</li>
    <li>libsForQt512.kimap</li>
    <li>libsForQt512.kio-extras</li>
    <li>libsForQt512.kio-gdrive</li>
    <li>libsForQt512.kipi-plugins</li>
    <li>libsForQt512.kitinerary</li>
    <li>libsForQt512.kldap</li>
    <li>libsForQt512.kleopatra</li>
    <li>libsForQt512.klettres</li>
    <li>libsForQt512.klines</li>
    <li>libsForQt512.kmag</li>
    <li>libsForQt512.kmahjongg</li>
    <li>libsForQt512.kmail</li>
    <li>libsForQt512.kmail-account-wizard</li>
    <li>libsForQt512.kmailtransport</li>
    <li>libsForQt512.kmbox</li>
    <li>libsForQt512.kmime</li>
    <li>libsForQt512.kmines</li>
    <li>libsForQt512.kmix</li>
    <li>libsForQt512.kmousetool</li>
    <li>libsForQt512.kmplot</li>
    <li>libsForQt512.knavalbattle</li>
    <li>libsForQt512.knetwalk</li>
    <li>libsForQt512.knights</li>
    <li>libsForQt512.knotes</li>
    <li>libsForQt512.koko</li>
    <li>libsForQt512.kolf</li>
    <li>libsForQt512.kollision</li>
    <li>libsForQt512.kolourpaint</li>
    <li>libsForQt512.kompare</li>
    <li>libsForQt512.konqueror</li>
    <li>libsForQt512.konquest</li>
    <li>libsForQt512.konsole</li>
    <li>libsForQt512.kontact</li>
    <li>libsForQt512.kontactinterface</li>
    <li>libsForQt512.korganizer</li>
    <li>libsForQt512.kpat</li>
    <li>libsForQt512.kpimtextedit</li>
    <li>libsForQt512.kpkpass</li>
    <li>libsForQt512.kpublictransport</li>
    <li>libsForQt512.kpurpose</li>
    <li>libsForQt512.kqtquickcharts</li>
    <li>libsForQt512.krdc</li>
    <li>libsForQt512.kreport</li>
    <li>libsForQt512.kreversi</li>
    <li>libsForQt512.krfb</li>
    <li>libsForQt512.kruler</li>
    <li>libsForQt512.kshisen</li>
    <li>libsForQt512.ksmtp</li>
    <li>libsForQt512.kspaceduel</li>
    <li>libsForQt512.ksquares</li>
    <li>libsForQt512.ksudoku</li>
    <li>libsForQt512.ksystemlog</li>
    <li>libsForQt512.kteatime</li>
    <li>libsForQt512.ktimer</li>
    <li>libsForQt512.ktnef</li>
    <li>libsForQt512.ktorrent</li>
    <li>libsForQt512.ktouch</li>
    <li>libsForQt512.ktrip</li>
    <li>libsForQt512.kturtle</li>
    <li>libsForQt512.kwalletmanager</li>
    <li>libsForQt512.kwave</li>
    <li>libsForQt512.libgravatar</li>
    <li>libsForQt512.libkcddb</li>
    <li>libsForQt512.libkdcraw</li>
    <li>libsForQt512.libkdegames</li>
    <li>libsForQt512.libkdepim</li>
    <li>libsForQt512.libkexiv2</li>
    <li>libsForQt512.libkgapi</li>
    <li>libsForQt512.libkipi</li>
    <li>libsForQt512.libkleo</li>
    <li>libsForQt512.libkmahjongg</li>
    <li>libsForQt512.libkomparediff2</li>
    <li>libsForQt512.libksane</li>
    <li>libsForQt512.libksieve</li>
    <li>libsForQt512.libktorrent</li>
    <li>libsForQt512.mailcommon</li>
    <li>libsForQt512.mailimporter</li>
    <li>libsForQt512.marble</li>
    <li>libsForQt512.mbox-importer</li>
    <li>libsForQt512.messagelib</li>
    <li>libsForQt512.minuet</li>
    <li>libsForQt512.okular</li>
    <li>libsForQt512.picmi</li>
    <li>libsForQt512.pim-data-exporter</li>
    <li>libsForQt512.pim-sieve-editor</li>
    <li>libsForQt512.pimcommon</li>
    <li>libsForQt512.plasma-browser-integration</li>
    <li>libsForQt512.plasma-desktop</li>
    <li>libsForQt512.print-manager</li>
    <li>libsForQt512.rocs</li>
    <li>libsForQt512.skanlite</li>
    <li>libsForQt512.skanpage</li>
    <li>libsForQt512.soundkonverter</li>
    <li>libsForQt512.spectacle</li>
    <li>libsForQt512.yakuake</li>
    <li>libsForQt512.zanshin</li>
    <li>libsForQt514.akonadi</li>
    <li>libsForQt514.akonadi-calendar</li>
    <li>libsForQt514.akonadi-calendar-tools</li>
    <li>libsForQt514.akonadi-contacts</li>
    <li>libsForQt514.akonadi-import-wizard</li>
    <li>libsForQt514.akonadi-mime</li>
    <li>libsForQt514.akonadi-notes</li>
    <li>libsForQt514.akonadi-search</li>
    <li>libsForQt514.akonadiconsole</li>
    <li>libsForQt514.akregator</li>
    <li>libsForQt514.angelfish</li>
    <li>libsForQt514.ark</li>
    <li>libsForQt514.baloo-widgets</li>
    <li>libsForQt514.bomber</li>
    <li>libsForQt514.bovo</li>
    <li>libsForQt514.calendarsupport</li>
    <li>libsForQt514.dolphin</li>
    <li>libsForQt514.dolphin-plugins</li>
    <li>libsForQt514.dragon</li>
    <li>libsForQt514.elisa</li>
    <li>libsForQt514.eventviews</li>
    <li>libsForQt514.falkon</li>
    <li>libsForQt514.ffmpegthumbs</li>
    <li>libsForQt514.filelight</li>
    <li>libsForQt514.granatier</li>
    <li>libsForQt514.grantleetheme</li>
    <li>libsForQt514.gwenview</li>
    <li>libsForQt514.incidenceeditor</li>
    <li>libsForQt514.k3b</li>
    <li>libsForQt514.kaccounts-integration</li>
    <li>libsForQt514.kaccounts-providers</li>
    <li>libsForQt514.kaddressbook</li>
    <li>libsForQt514.kalarm</li>
    <li>libsForQt514.kalendar</li>
    <li>libsForQt514.kalzium</li>
    <li>libsForQt514.kamoso</li>
    <li>libsForQt514.kapman</li>
    <li>libsForQt514.kapptemplate</li>
    <li>libsForQt514.kate</li>
    <li>libsForQt514.katomic</li>
    <li>libsForQt514.kblackbox</li>
    <li>libsForQt514.kblocks</li>
    <li>libsForQt514.kbounce</li>
    <li>libsForQt514.kbreakout</li>
    <li>libsForQt514.kcachegrind</li>
    <li>libsForQt514.kcalc</li>
    <li>libsForQt514.kcalutils</li>
    <li>libsForQt514.kcharselect</li>
    <li>libsForQt514.kcolorchooser</li>
    <li>libsForQt514.kdebugsettings</li>
    <li>libsForQt514.kdeconnect-kde</li>
    <li>libsForQt514.kdegraphics-mobipocket</li>
    <li>libsForQt514.kdegraphics-thumbnailers</li>
    <li>libsForQt514.kdenetwork-filesharing</li>
    <li>libsForQt514.kdenlive</li>
    <li>libsForQt514.kdepim-addons</li>
    <li>libsForQt514.kdepim-runtime</li>
    <li>libsForQt514.kdeplasma-addons</li>
    <li>libsForQt514.kdf</li>
    <li>libsForQt514.kdialog</li>
    <li>libsForQt514.kdiamond</li>
    <li>libsForQt514.keditbookmarks</li>
    <li>libsForQt514.kfind</li>
    <li>libsForQt514.kfloppy</li>
    <li>libsForQt514.kgeography</li>
    <li>libsForQt514.kget</li>
    <li>libsForQt514.kgpg</li>
    <li>libsForQt514.khelpcenter</li>
    <li>libsForQt514.kidentitymanagement</li>
    <li>libsForQt514.kig</li>
    <li>libsForQt514.kigo</li>
    <li>libsForQt514.killbots</li>
    <li>libsForQt514.kimap</li>
    <li>libsForQt514.kio-extras</li>
    <li>libsForQt514.kio-gdrive</li>
    <li>libsForQt514.kipi-plugins</li>
    <li>libsForQt514.kitinerary</li>
    <li>libsForQt514.kldap</li>
    <li>libsForQt514.kleopatra</li>
    <li>libsForQt514.klettres</li>
    <li>libsForQt514.klines</li>
    <li>libsForQt514.kmag</li>
    <li>libsForQt514.kmahjongg</li>
    <li>libsForQt514.kmail</li>
    <li>libsForQt514.kmail-account-wizard</li>
    <li>libsForQt514.kmailtransport</li>
    <li>libsForQt514.kmbox</li>
    <li>libsForQt514.kmime</li>
    <li>libsForQt514.kmines</li>
    <li>libsForQt514.kmix</li>
    <li>libsForQt514.kmousetool</li>
    <li>libsForQt514.kmplot</li>
    <li>libsForQt514.knavalbattle</li>
    <li>libsForQt514.knetwalk</li>
    <li>libsForQt514.knights</li>
    <li>libsForQt514.knotes</li>
    <li>libsForQt514.koko</li>
    <li>libsForQt514.kolf</li>
    <li>libsForQt514.kollision</li>
    <li>libsForQt514.kolourpaint</li>
    <li>libsForQt514.kompare</li>
    <li>libsForQt514.konqueror</li>
    <li>libsForQt514.konquest</li>
    <li>libsForQt514.konsole</li>
    <li>libsForQt514.kontact</li>
    <li>libsForQt514.kontactinterface</li>
    <li>libsForQt514.korganizer</li>
    <li>libsForQt514.kpat</li>
    <li>libsForQt514.kpimtextedit</li>
    <li>libsForQt514.kpkpass</li>
    <li>libsForQt514.kpublictransport</li>
    <li>libsForQt514.kpurpose</li>
    <li>libsForQt514.kqtquickcharts</li>
    <li>libsForQt514.krdc</li>
    <li>libsForQt514.kreport</li>
    <li>libsForQt514.kreversi</li>
    <li>libsForQt514.krfb</li>
    <li>libsForQt514.kruler</li>
    <li>libsForQt514.kshisen</li>
    <li>libsForQt514.ksmtp</li>
    <li>libsForQt514.kspaceduel</li>
    <li>libsForQt514.ksquares</li>
    <li>libsForQt514.ksudoku</li>
    <li>libsForQt514.ksystemlog</li>
    <li>libsForQt514.kteatime</li>
    <li>libsForQt514.ktimer</li>
    <li>libsForQt514.ktnef</li>
    <li>libsForQt514.ktorrent</li>
    <li>libsForQt514.ktouch</li>
    <li>libsForQt514.ktrip</li>
    <li>libsForQt514.kturtle</li>
    <li>libsForQt514.kwalletmanager</li>
    <li>libsForQt514.kwave</li>
    <li>libsForQt514.libgravatar</li>
    <li>libsForQt514.libkcddb</li>
    <li>libsForQt514.libkdcraw</li>
    <li>libsForQt514.libkdegames</li>
    <li>libsForQt514.libkdepim</li>
    <li>libsForQt514.libkexiv2</li>
    <li>libsForQt514.libkgapi</li>
    <li>libsForQt514.libkipi</li>
    <li>libsForQt514.libkleo</li>
    <li>libsForQt514.libkmahjongg</li>
    <li>libsForQt514.libkomparediff2</li>
    <li>libsForQt514.libksane</li>
    <li>libsForQt514.libksieve</li>
    <li>libsForQt514.libktorrent</li>
    <li>libsForQt514.mailcommon</li>
    <li>libsForQt514.mailimporter</li>
    <li>libsForQt514.marble</li>
    <li>libsForQt514.mbox-importer</li>
    <li>libsForQt514.messagelib</li>
    <li>libsForQt514.minuet</li>
    <li>libsForQt514.okular</li>
    <li>libsForQt514.picmi</li>
    <li>libsForQt514.pim-data-exporter</li>
    <li>libsForQt514.pim-sieve-editor</li>
    <li>libsForQt514.pimcommon</li>
    <li>libsForQt514.plasma-browser-integration</li>
    <li>libsForQt514.plasma-desktop</li>
    <li>libsForQt514.print-manager</li>
    <li>libsForQt514.rocs</li>
    <li>libsForQt514.skanlite</li>
    <li>libsForQt514.skanpage</li>
    <li>libsForQt514.soundkonverter</li>
    <li>libsForQt514.spectacle</li>
    <li>libsForQt514.yakuake</li>
    <li>libsForQt514.zanshin</li>
  </ul>
</details>
<details>
  <summary>183 packages built:</summary>
  <ul>
    <li>ajour</li>
    <li>calligra</li>
    <li>digikam</li>
    <li>haruna</li>
    <li>kdev-php</li>
    <li>kdev-python</li>
    <li>kdevelop</li>
    <li>kdevelop-unwrapped</li>
    <li>kile</li>
    <li>kmymoney</li>
    <li>libsForQt5.akonadi</li>
    <li>libsForQt5.akonadi-calendar</li>
    <li>libsForQt5.akonadi-calendar-tools</li>
    <li>libsForQt5.akonadi-contacts</li>
    <li>libsForQt5.akonadi-import-wizard</li>
    <li>libsForQt5.akonadi-mime</li>
    <li>libsForQt5.akonadi-notes</li>
    <li>libsForQt5.akonadi-search</li>
    <li>libsForQt5.akonadiconsole</li>
    <li>libsForQt5.akregator</li>
    <li>libsForQt5.angelfish</li>
    <li>libsForQt5.ark</li>
    <li>libsForQt5.baloo-widgets</li>
    <li>libsForQt5.bomber</li>
    <li>libsForQt5.bovo</li>
    <li>libsForQt5.calendarsupport</li>
    <li>libsForQt5.dolphin</li>
    <li>libsForQt5.dolphin-plugins</li>
    <li>libsForQt5.dragon</li>
    <li>libsForQt5.elisa</li>
    <li>libsForQt5.eventviews</li>
    <li>libsForQt5.falkon</li>
    <li>libsForQt5.ffmpegthumbs</li>
    <li>libsForQt5.filelight</li>
    <li>libsForQt5.granatier</li>
    <li>libsForQt5.grantleetheme</li>
    <li>libsForQt5.gwenview</li>
    <li>libsForQt5.incidenceeditor</li>
    <li>libsForQt5.k3b</li>
    <li>libsForQt5.kaccounts-integration</li>
    <li>libsForQt5.kaccounts-providers</li>
    <li>libsForQt5.kaddressbook</li>
    <li>libsForQt5.kalarm</li>
    <li>libsForQt5.kalendar</li>
    <li>libsForQt5.kalzium</li>
    <li>libsForQt5.kamoso</li>
    <li>libsForQt5.kapman</li>
    <li>libsForQt5.kapptemplate</li>
    <li>libsForQt5.kate</li>
    <li>libsForQt5.katomic</li>
    <li>libsForQt5.kblackbox</li>
    <li>libsForQt5.kblocks</li>
    <li>libsForQt5.kbounce</li>
    <li>libsForQt5.kbreakout</li>
    <li>libsForQt5.kcachegrind</li>
    <li>libsForQt5.kcalc</li>
    <li>libsForQt5.kcalutils</li>
    <li>libsForQt5.kcharselect</li>
    <li>libsForQt5.kcolorchooser</li>
    <li>libsForQt5.kdebugsettings</li>
    <li>libsForQt5.kdeconnect-kde</li>
    <li>libsForQt5.kdegraphics-mobipocket</li>
    <li>libsForQt5.kdegraphics-thumbnailers</li>
    <li>libsForQt5.kdenetwork-filesharing</li>
    <li>libsForQt5.kdenlive</li>
    <li>libsForQt5.kdepim-addons</li>
    <li>libsForQt5.kdepim-runtime</li>
    <li>libsForQt5.kdeplasma-addons</li>
    <li>libsForQt5.kdf</li>
    <li>libsForQt5.kdialog</li>
    <li>libsForQt5.kdiamond</li>
    <li>libsForQt5.keditbookmarks</li>
    <li>libsForQt5.kfind</li>
    <li>libsForQt5.kfloppy</li>
    <li>libsForQt5.kgeography</li>
    <li>libsForQt5.kget</li>
    <li>libsForQt5.kgpg</li>
    <li>libsForQt5.khelpcenter</li>
    <li>libsForQt5.kidentitymanagement</li>
    <li>libsForQt5.kig</li>
    <li>libsForQt5.kigo</li>
    <li>libsForQt5.killbots</li>
    <li>libsForQt5.kimap</li>
    <li>libsForQt5.kio-extras</li>
    <li>libsForQt5.kio-gdrive</li>
    <li>libsForQt5.kipi-plugins</li>
    <li>libsForQt5.kitinerary</li>
    <li>libsForQt5.kldap</li>
    <li>libsForQt5.kleopatra</li>
    <li>libsForQt5.klettres</li>
    <li>libsForQt5.klines</li>
    <li>libsForQt5.kmag</li>
    <li>libsForQt5.kmahjongg</li>
    <li>libsForQt5.kmail</li>
    <li>libsForQt5.kmail-account-wizard</li>
    <li>libsForQt5.kmailtransport</li>
    <li>libsForQt5.kmbox</li>
    <li>libsForQt5.kmime</li>
    <li>libsForQt5.kmines</li>
    <li>libsForQt5.kmix</li>
    <li>libsForQt5.kmousetool</li>
    <li>libsForQt5.kmplot</li>
    <li>libsForQt5.knavalbattle</li>
    <li>libsForQt5.knetwalk</li>
    <li>libsForQt5.knights</li>
    <li>libsForQt5.knotes</li>
    <li>libsForQt5.koko</li>
    <li>libsForQt5.kolf</li>
    <li>libsForQt5.kollision</li>
    <li>libsForQt5.kolourpaint</li>
    <li>libsForQt5.kompare</li>
    <li>libsForQt5.konqueror</li>
    <li>libsForQt5.konquest</li>
    <li>libsForQt5.konsole</li>
    <li>libsForQt5.kontact</li>
    <li>libsForQt5.kontactinterface</li>
    <li>libsForQt5.korganizer</li>
    <li>libsForQt5.kpat</li>
    <li>libsForQt5.kpimtextedit</li>
    <li>libsForQt5.kpkpass</li>
    <li>libsForQt5.kpublictransport</li>
    <li>libsForQt5.kpurpose</li>
    <li>libsForQt5.kqtquickcharts</li>
    <li>libsForQt5.krdc</li>
    <li>libsForQt5.kreport</li>
    <li>libsForQt5.kreversi</li>
    <li>libsForQt5.krfb</li>
    <li>libsForQt5.kruler</li>
    <li>libsForQt5.kshisen</li>
    <li>libsForQt5.ksmtp</li>
    <li>libsForQt5.kspaceduel</li>
    <li>libsForQt5.ksquares</li>
    <li>libsForQt5.ksudoku</li>
    <li>libsForQt5.ksystemlog</li>
    <li>libsForQt5.kteatime</li>
    <li>libsForQt5.ktimer</li>
    <li>libsForQt5.ktnef</li>
    <li>libsForQt5.ktorrent</li>
    <li>libsForQt5.ktouch</li>
    <li>libsForQt5.ktrip</li>
    <li>libsForQt5.kturtle</li>
    <li>libsForQt5.kwalletmanager</li>
    <li>libsForQt5.kwave</li>
    <li>libsForQt5.libgravatar</li>
    <li>libsForQt5.libkcddb</li>
    <li>libsForQt5.libkdcraw</li>
    <li>libsForQt5.libkdegames</li>
    <li>libsForQt5.libkdepim</li>
    <li>libsForQt5.libkexiv2</li>
    <li>libsForQt5.libkgapi</li>
    <li>libsForQt5.libkipi</li>
    <li>libsForQt5.libkleo</li>
    <li>libsForQt5.libkmahjongg</li>
    <li>libsForQt5.libkomparediff2</li>
    <li>libsForQt5.libksane</li>
    <li>libsForQt5.libksieve</li>
    <li>libsForQt5.libktorrent</li>
    <li>libsForQt5.mailcommon</li>
    <li>libsForQt5.mailimporter</li>
    <li>libsForQt5.marble</li>
    <li>libsForQt5.mbox-importer</li>
    <li>libsForQt5.messagelib</li>
    <li>libsForQt5.minuet</li>
    <li>libsForQt5.okular</li>
    <li>libsForQt5.picmi</li>
    <li>libsForQt5.pim-data-exporter</li>
    <li>libsForQt5.pim-sieve-editor</li>
    <li>libsForQt5.pimcommon</li>
    <li>libsForQt5.plasma-browser-integration</li>
    <li>libsForQt5.plasma-desktop</li>
    <li>libsForQt5.print-manager</li>
    <li>libsForQt5.rocs</li>
    <li>libsForQt5.skanlite</li>
    <li>libsForQt5.skanpage</li>
    <li>libsForQt5.soundkonverter</li>
    <li>libsForQt5.spectacle</li>
    <li>libsForQt5.yakuake</li>
    <li>libsForQt5.zanshin</li>
    <li>qcachegrind</li>
    <li>tellico</li>
    <li>trojita</li>
    <li>virt-manager-qt</li>
    <li>zombietrackergps</li>
  </ul>
</details>

**Comment by andrevmatos** [comment]
@SuperSandro2000 could you, please, give a look at this PR? Should be trivial.

**Comment by andrevmatos** [comment]
cc @NixOS/qt-kde

**Comment by peterhoeg** [comment]
Should we target staging instead considering the frameworks bump is going in there?

**Comment by andrevmatos** [comment]
@peterhoeg I don't think so. That staging kf5 bump may gonna take weeks to arrive to master, and since this is just a patch bump, and it includes the fix for calligra on poppler 22.04, which is currently broken on master, I'd think this is better in master before the staging rebuild.

File: pkgs/applications/office/calligra/default.nix [hunk 1]
```diff
@@ -30,18 +30,23 @@ mkDerivation rec {
...
     # Fixes for building calligra with gcc11/c++17
     (fetchpatch {
       name = "build_c++17_poppler.patch";
       url = "https://github.com/archlinux/svntogit-packages/raw/bbbe35f97eb1033798f1cf95d427890168598199/trunk/068cd9ae.patch";
       sha256 = "sha256-d9/ILwSeW+ov11DF191hzIaUafO/rjQrAeONwqDSKbA=";
     })
+    # Fixes for building calligra with modern poppler[-qt5]
+    (fetchpatch {
+      name = "calligra-poppler-22.03.patch";
+      url = "https://invent.kde.org/office/calligra/-/commit/236bacbe.patch";
```
Comment (SuperSandro2000) [inline pkgs/applications/office/calligra/default.nix:42]:
```suggestion
      url = "https://invent.kde.org/office/calligra/-/commit/236bacbe13739414e919de868283b0caf2df5d8a.patch";
```

File: pkgs/applications/office/calligra/default.nix [hunk 2]
```diff
@@ -30,18 +30,23 @@ mkDerivation rec {
...
+      sha256 = "sha256-9DmKPCvEFy2Cs3g7350iOCF5Vrx1HL+/8jr+Tb44CyE=";
+    })
+    (fetchpatch {
+      name = "calligra-poppler-22.04.patch";
+      url = "https://invent.kde.org/office/calligra/-/commit/6b75bec7.patch";
```
Comment (SuperSandro2000) [inline pkgs/applications/office/calligra/default.nix:47]:
```suggestion
      url = "https://invent.kde.org/office/calligra/-/commit/6b75bec784c9835c78993349845d8c2ef22ec3de.patch";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Short shas can be DOSed.

File: pkgs/applications/office/calligra/default.nix (hunk 1 above)
Comment (andrevmatos) [reply pkgs/applications/office/calligra/default.nix]: Done; amended to avoid commit history spam

**Pull request merged**